source/configuration_manual/stats/index.rst

```diff
@@ -21,6 +21,8 @@ In addition to the event filter, a list of fields that are included in the
 metrics can be specified using the ``fields`` setting.  All events have a
 default "duration" field that doesn't need to be listed explicitly.
 
+.. versionchanged:: v2.4.0,v3.0.0 This now applies to OpenMetrics too.
```

Comment:
Pretty unclear what is "This"?

Reply:
the fields setting, i can of course repeat it there.

Reply:
I think a standalone sentence makes more sense, since otherwise it's referring to a part of an earlier paragraph.